### DISCUSSION TITLE
[Feature] add to favorite images with partner sharing

### DISCUSSION THREAD

**Discussion by azertylr** [immich-app/immich#12902] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Hello,

It would be nice if we can add to our favorites, image that have been shared through partner sharing :)

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by danieldietzler** [comment]
Covered by #12614
